Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/GlideUtils.java

```diff
@@ -51,102 +51,21 @@ public static void loadGameBanner(ImageView imageView, GameFile gameFile)
   public static void loadGameCover(ImageView imageView, GameFile gameFile)
   {
     Context context = imageView.getContext();
-    File cover = new File(gameFile.getCustomCoverPath());
```

Comment:
Please preserve the ability to set custom cover art for games.

Reply:
Ah, I didn't even realize we had that feature. I'll fix it and test it appropriately.